app/src/main/res/layout/list_item_drawer.xml

```diff
@@ -65,7 +65,6 @@
             android:layout_marginRight="16dp"
             android:fontFamily="sans-serif"
             android:gravity="center"
-            android:textColor="?colorPrimary"
```

Comment:
I assume this is commented out to make the nav drawer text white?

Reply:
Ahh, yes. I think this is the tag that changes the sleep timer `editText` to the primary color to be precise.